lib/nimble_template/helpers/generator.ex

```diff
@@ -40,7 +40,21 @@ defmodule NimbleTemplate.Generator do
         File.write!(file, [left, content, right])
 
       :error ->
-        Mix.raise(~s[Could not find #{anchor} in #{file_path}])
+        if raise_exception do
+          Mix.raise(~s[Could not find #{anchor} in #{file_path}])
+        else
+          :error
+        end
+    end
+  end
+
+  def replace_content_all(file_path, anchor, content) do
+    case replace_content(file_path, anchor, content, false) do
```

Comment:
Minor comment. Looking at this line it is unclear what exactly `false` arg is intended for. We would need to check the function definition to find out that. Maybe tweak it like this? Since there are no named arguments in Elixir (well there are, but they requires to use maps).
```suggestion
    raise_exception = false
    case replace_content(file_path, anchor, content, raise_exception) do
```

Reply:
Hi @Nihisil with the @olivierobert suggestion above, we will have separate method

1. replace_content -> which won't raise any exception
2. replace_content! -> which will raise the exception

So we will remove the `false` argument here, I planed to refactor it on the different PR, is that ok?

Reply:
Sure, it is totally fine. Thanks!

Reply:
Noted ✅